Makefile

```diff
@@ -381,7 +380,7 @@ oidc-discovery-provider-scratch-image: Dockerfile
 #############################################################################
 
 .PHONY: images-windows
-images-windows: spire-server-image-windows spire-agent-image-windows k8s-workload-registrar-image-windows oidc-discovery-provider-image-windows
+images-windows: spire-server-image-windows spire-agent-image-windows oidc-discovery-provider-image-windows
```

Comment:
what do you think about remove `k8s-workload-registrar-image-windows`  from `images-windows` 
but keep `make k8s-workload-registrar-image-windows`?

Reply:
Sounds good!